lib/ops/opsservice/report.go

```diff
@@ -341,6 +347,30 @@ func collectOperationLogs(site site, operation ops.SiteOperation, reportWriter r
 	return trace.Wrap(err)
 }
 
+// collectStatusHistory collects the cluster status history and stores it in the
+// specified dir.
+func collectStatusHistory(dir string) error {
```

Comment:
I'm a bit on the fence with this being implemented on cluster controller - what if the controller is inaccessible but the timeline API is? This can be equally well moved to the `lib/report` package which this package uses so it should be win/win.

Reply:
I think this is a good point. @bernardjkim - let's do it like this.

Reply:
Updated in 1b46f3e